src/components/RenderPost.vue

```diff
@@ -11,7 +11,13 @@
 
 <script>
 export default {
-    props:["title", "body", "postId", "activePage"]
+    props:["title", "body", "postId", "activePage", "sortOrder"],
+
+    methods:{
+      storeSortOrder(){
+        sessionStorage.setItem("sortOrder", String(this.sortOrder));
+      }
+    }
```

Comment:
@ChoudharyHamza looks like you are storing the sort order in session storage when user click on the article title. Instead store it when user click on any sort button. It should not be dependent on article clicking.

Reply:
ok